static/js/components/CompactCourseDisplay.js

```diff
@@ -0,0 +1,69 @@
+// @flow
+/* global SETTINGS:false */
+import React from "react"
+import { connect } from "react-redux"
+
+import Card from "./Card"
+import { courseAvailability, maxPrice } from "../lib/courses"
+import { embedlyThumbnail } from "../lib/url"
+import { EMBEDLY_THUMB_HEIGHT, EMBEDLY_THUMB_WIDTH } from "../lib/posts"
+
+import type { Course } from "../flow/discussionTypes"
+
+type Props = {
+  course: Course
+}
+
+export class CompactCourseDisplay extends React.Component<Props> {
+  render() {
+    const { course } = this.props
+    return (
+      <Card className={`compact-course-summary`}>
+        <div className="column1">
+          <div className="preview-body">
+            <div className="row title-row">
+              <div className="course-title">{course.title}</div>
+            </div>
+            <div className="row">
+              <div className="course-topics">
+                {course.topics ? course.topics[0].name : ""}
```

Comment:
Are topics in any particular order? Does it matter?

Reply:
They are not in any particular order.  @Ferdi @pdpinch do you want the topics in any particular order, such as alphabetical?  All topics will be shown on the detail page.